index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<script defer src="src/Script.js"></script>
```

Comment:
I was getting the below message from parcel
```suggestion
	<script type="module" defer defer src="src/Script.js"></script>
```
![image](https://user-images.githubusercontent.com/21975874/220301107-c5ab2700-3e1d-4940-9796-6b6cd7516ba7.png)

Reply:
@omedusyo you might have the same problem

Reply:
did you install `parcel` or `parcel-bundler`? I at first installed `parcel`, but after following instructions more carefuly it works for me :D

Reply:
I followed the instructions i the README and now checked my history to be sure and I installed parcel-bundler (but maybe I already had parcel and it has precedence 🤷 )